src/main_app/config/classes.py

```
@@ -92,16 +92,26 @@ class Settings:
     csrf_time_limit: int | None  # None means never expire
+
+
+__all__ = [
+    "DbConfig",
+    "Paths",
+    "CookieConfig",
+    "SessionConfig",
+    "OAuthConfig",
+    "CorsConfig",
+    "Settings",
+]
```

Ruff (0.15.13)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)